### PR TITLE
Annotation was unusable

### PR DESCRIPTION
**Pull request by Bantolomeus**

### CODE REVIEW

**Comment by Bantolomeus** [comment]
@Husterknupp can I merge my branch into master? Wanted you to check it, just in case.

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
-        <kotlin.version>1.3.20</kotlin.version>
+        <kotlin.version>1.6.10</kotlin.version>
```
Comment (Husterknupp) [inline pom.xml:28]: please update the version in the readme accordingly

File: pom.xml [hunk 2]
```diff
@@ -72,6 +72,22 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-test</artifactId>
+            <version>${kotlin.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jdk8</artifactId>
```
Comment (Husterknupp) [inline pom.xml:88]: is this not a duplicate entry? See two dependencies up

**Review by Husterknupp — CHANGES_REQUESTED** [review]
Sure. Please address the two notes first and then merge into master

**Review by Husterknupp — APPROVED** [review]
lgtm

**Pull request merged**